lib/puppet/type/cisco_aci_vzentry.rb

```diff
@@ -66,62 +68,62 @@ def self.title_patterns
   # ---------------------------------------------------------------
 
   newproperty(:apply_to_frag) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#applyToFrag'
+    desc ''
   end
 
   newproperty(:arp_opc) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#arpOpc'
+    desc 'ARP opcodes.'
   end
 
   newproperty(:d_from_port) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#dFromPort'
+    desc 'Destination From Port.'
   end
 
   newproperty(:d_to_port) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#dToPort'
+    desc 'Destination To Port.'
   end
 
   newproperty(:descr) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#descr'
+    desc 'Specifies the description of a policy component.'
   end
 
   newproperty(:ether_t) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#etherT'
+    desc 'Ether type.'
   end
 
   newproperty(:icmpv4_t) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#icmpv4T'
+    desc ''
   end
 
   newproperty(:icmpv6_t) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzEntry.html#icmpv6T'
+    desc ''
```

Comment:
Again here.  We could simply fill in `Specify ICMP v6 type` or something simple along those lines.